### DISCUSSION TITLE
How can I produce the sums of total points in "voxel pillars"?

### DISCUSSION THREAD

**Discussion by mavavilj** [r-lidar/lidR#441] [Q&A]
How can I produce the sums of total points in "voxel pillars"?

Intuitively I started with `voxel_metrics(las, ~length(Z),`

But after this I'd need to sum along Z-axis on each X-Y "block".

And I haven't been able to figure out how to do this effectively.

(sorry if this forum isn't for these questions)

**Comment by Jean-Romain** [comment]
`grid_metrics()` ? 

Your question is unclear. Please asked a clear and well defined question on [gis.stackexchange](https://gis.stackexchange.com/questions/tagged/lidr) with the tag `lidr`
